hal/src/thumbv6m/clock.rs

```diff
@@ -321,7 +338,7 @@ impl GenericClockController {
             XOSC32K | OSC32K | OSCULP32K => OSC32K_FREQ,
             GCLKGEN1 => self.gclks[1],
             OSC8M => OSC8M_FREQ,
-            DFLL48M => OSC48M_FREQ,
+            ClockSource::DFLL48M => OSC48M_FREQ,
```

Comment:
why did you add this prefix?

Reply:
In the SAMD20, the PAC has `pac::gclk::clkctrl::ID_A::DFLL48M` and `pac::gclk::genctrl::SRC_A::DFLL48M`. As ID_A and SRC_A are both glob-imported, that was making a ambiguous reference error